hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/TestContainerReportHandler.java

```diff
@@ -100,7 +101,6 @@ void setup() throws IOException, InvalidStateTransitionException {
     dbStore = DBStoreBuilder.createDBStore(
         conf, new SCMDBDefinition());
     scmhaManager = SCMHAManagerStub.getInstance(true);
-    nodeManager = new MockNodeManager(true, 10);
```

Comment:
Why was this line removed?

Reply:
It was a duplicate, the same statement is also present a couple lines above: https://github.com/apache/ozone/blob/a6b3392c0718761a0936e267c86b5d84d81c1072/hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/TestContainerReportHandler.java#L98